test/groovy/BaseTest.groovy

```diff
@@ -1,10 +1,8 @@
 import com.lesfurets.jenkins.unit.declarative.DeclarativePipelineTest
 import mock.CurrentBuild
-import mock.CustomWARPackager
 import mock.Infra
 import org.junit.Before
 import org.junit.Test
-import org.yaml.snakeyaml.Yaml
```

Comment:
Is https://github.com/jenkins-infra/pipeline-library/blob/cd87a074cbdcdb3072f7ee651afa5db7c8fdba40/pom.xml#L62-L67 still needed after the deletion of all its callers?